src/Panel.js

```diff
@@ -99,6 +99,7 @@ class Panel extends React.Component {
         role={role}
         href={id && `#${id}`}
         onClick={this.handleClickTitle}
+        className={ expanded ? '' : 'collapsed' }
```

Comment:
can we use `classNames(expanded && 'collapsed')`? it'll prevent having an empty `class` attribute.